packages/cron/src/jobs/pins.js

```diff
@@ -63,9 +64,22 @@ export async function updatePinStatuses ({ cluster, db }) {
         return null // not tracked by our cluster
       }
 
-      const status = toPinStatusEnum(peerMap[pin.location.peerId].status)
+      let status = toPinStatusEnum(peerMap[pin.location.peerId].status)
 
       if (status !== 'Pinned' && status !== 'Remote') {
```

Comment:
If the status is `Unpinned` that's when we need to try the other version.

THEN we need this check (`if (status !== 'Pinned' && status !== 'Remote') {`) as it was before.